privaterelay/views.py

```diff
@@ -275,10 +277,21 @@ def _update_all_data(social_account, extra_data, new_email):
             newly_premium = not had_premium and now_has_premium
             no_longer_premium = had_premium and not now_has_premium
             if newly_premium:
+                incr_if_enabled("user_purchased_premium", 1)
```

Comment:
note: this fixes a long-standing issue where these aren't showing up in grafana.